schema.sql

```diff
@@ -54,12 +54,15 @@ CREATE TABLE specializations (
     FOREIGN kEY (vets_id) REFERENCES vets(id),
     PRIMARY kEY (species_id, vets_id)
 );
-CREATE TABLE visits (
-    animals_id INT,
-    vets_id INT,
-    date_of_visit DATE,
-    id INT GENERATED ALWAYS AS IDENTITY,
-    FOREIGN KEY (animals_id) REFERENCES animals(id),
-    FOREIGN KEY (vets_id) REFERENCES vets(id),
-    PRIMARY KEY (id)
-);
\ No newline at end of file
+
+
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+SELECT COUNT(*) FROM visits where animal_id = 4;
+SELECT * FROM visits where vet_id = 2;
+SELECT * FROM owners where email = 'owner_18327@mail.com';
```

Comment:
- [ ] please make sure this only has schema to create tables and change tables, therefor you should move the select queries to `queries.sql` in order to keep the file clean and organized